### DISCUSSION TITLE
downloading pdf files

### DISCUSSION THREAD

**Discussion by BilleBarrett** [sandboxie-plus/Sandboxie#1487] [General]
Release v1.0.5
I noticed today that if I download a pdf file, the recover windows doesn't open I have to recover
I have to right-click on the Plus icon, then right-click on the default icon, then left-click on recover then do the recovery.
Is there something I need to declare to have this happen, it seems to be only on PDF files
Thank you
Bill

**Comment by isaak654** [comment]
> Release v1.0.5 I noticed today that if I download a pdf file, the recover windows doesn't open I have to recover I have to right-click on the Plus icon, then right-click on the default icon, then left-click on recover then do the recovery. Is there something I need to declare to have this happen, it seems to be only on PDF files Thank you Bill

I'm unable to reproduce it on W10 with 1.0.5.
`AutoRecover=y` was already set previously.

**Comment by BilleBarrett** [comment]
Went to read the BBB bill @ https://www.congress.gov/bill/117th-congress/house-bill/5376/text

I ten click on the pdf (4mb) it appears at the bottom Chrome and with Bills name I clear the download at the bottom 

I then cleared all windows and looked for the Recovery window which is usually there, it was not 
I then had to follow the procedure I posted to recover the pdf

Text available as:

*	 <https://www.congress.gov/117/bills/hr5376/generated/BILLS-117hr5376rh.html> XML/HTML (5MB)
*	TXT (3MB)
*	 <https://www.congress.gov/117/bills/hr5376/BILLS-117hr5376rh.pdf> PDF (4MB) (PDF provides a complete and accurate display of this text.) Tip?

This text has been loaded in plain text format due to the large size of the XML/HTML file. Loading the  <https://www.congress.gov/117/bills/hr5376/BILLS-117hr5376rh.xml> XML/HTML in a new window (4MB) may take several minutes or possibly cause your browser to become unresponsive.

 

 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891   ***@***.***
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Reply by isaak654** [reply]
> Went to read the BBB bill @ https://www.congress.gov/bill/117th-congress/house-bill/5376/text I ten click on the pdf (4mb) it appears at the bottom Chrome and with Bills name I clear the download at the bottom I then cleared all windows and looked for the Recovery window which is usually there, it was not I then had to follow the procedure I posted to recover the pdf

The pdf file appears correctly for me in the recovery window after clicking on the `PDF (4MB)` link and saving the pdf file from a sandboxed Chrome session (I don't keep any extension there).
